app/controllers/v0/in_progress_forms_controller.rb

```diff
@@ -26,14 +26,10 @@ def update
 
       form.update!(form_data: params[:form_data] || params[:formData], metadata: params[:metadata])
 
-      veteran_icn = @current_user.icn
       if Flipper.enabled?(:intent_to_file_lighthouse_enabled, @current_user) && form.id_previously_changed? &&
-         Lighthouse::CreateIntentToFileJob::ITF_FORMS.include?(form.form_id) && veteran_icn.present?
-        if @current_user.participant_id.blank?
-          track_missing_user_pids(form)
-        else
-          Lighthouse::CreateIntentToFileJob.perform_async(form.form_id, form.created_at, veteran_icn)
-        end
+         Lighthouse::CreateIntentToFileJob::ITF_FORMS.include?(form.form_id)
```

Comment:
This is getting ugly.  When we take out the flipper it will be better.

Reply:
i could also just remove the flipper in this PR as well. This is the only place in the backend that we reference this flipper.

Reply:
Let's do it separately so this PR isn't more than it is.